examples/simulation.py

```diff
@@ -18,13 +18,18 @@ def __init__(self, ipdb):
     # if called directly outside Simulation class, "ifc_base_name" should be
     # different from "name", the "ifc_base_name" and "name" are the same for
     # the first ifc created by namespace
-    def _ns_add_ifc(self, name, ns_ifc, ifc_base_name, in_ifc=None, out_ifc=None,
-                    ipaddr=None, macaddr=None, fn=None, cmd=None, action="ok",
-                    disable_ipv6=False):
+    def _ns_add_ifc(self, name, ns_ifc, ifc_base_name=None, in_ifc=None,
+                    out_ifc=None, ipaddr=None, macaddr=None, fn=None, cmd=None,
+                    action="ok", disable_ipv6=False):
         if name in self.ipdbs:
             ns_ipdb = self.ipdbs[name]
         else:
             ns_ipdb = IPDB(nl=NetNS(name))
+            if disable_ipv6:
+                cmd = ["sysctl", "-q", "-w", "net.ipv6.conf.default.disable_ipv6=1"]
+                nsp = NSPopen(ns_ipdb.nl.netns, cmd)
+                nsp.wait(); nsp.release()
+            ns_ipdb.interfaces.lo.up().commit()
```

Comment:
I see. You are using a bridge to link the two veth pairs together. It could be done by inserting one end of a veth pair into the other namespace directly, such that neither of them show up in the abstract namespace. In fact, the support for this is exactly what this function is supposed to achieve, but in a different set of code example (in a private codebase, so you don't see it here). I will amend the patch to use that approach, and eliminate the extra bridges.